test/deploy-cluster.sh

```diff
@@ -42,7 +42,8 @@ function clean_up {
   for POD_NAME in "${ALL_PODS[@]}"; do
     pod_info_file="$POD_INFO_DIR/$POD_NAME.txt"
     echo "Pod name: $POD_NAME" >> "$pod_info_file"
-    echo "Detailed logs: https://console.cloud.google.com/logs/viewer?project=$PROJECT&advancedFilter=resource.type%3D%22k8s_container%22%0Aresource.labels.project_id%3D%22$PROJECT%22%0Aresource.labels.location%3D%22us-east1-b%22%0Aresource.labels.cluster_name%3D%22${TEST_CLUSTER}%22%0Aresource.labels.namespace_name%3D%22$NAMESPACE%22%0Aresource.labels.pod_name%3D%22$POD_NAME%22" \
+    echo "Detailed logs:" >> "$pod_info_file"
```

Comment:
Maybe you can also dump logs as separate files using `kubectl log`

Reply:
I just thought there will be too much, and I need to dump log for each container, and log could contain credentials... so I decided to keep them on stackdriver.